src/common/setup.rs

```diff
@@ -68,9 +72,26 @@ pub fn find_default_config_dir() -> Option<PathBuf> {
     None
 }
 
+/// Looks for an existing dir, uses that, else returns a
+/// dir matching the config spec.
+pub fn get_default_data_dir() -> PathBuf {
+    vec![
+        Some(xdg_data_dir()),
+        Some(Path::new(SYSTEM_DEFAULT_DATA_DIR_PREFIX).join("/share/zellij")),
+    ]
+    .into_iter()
+    .filter(|p| p.is_some())
```

Comment:
Any reason this is needed? (I assume it's a form of future-proofing? When will these values be `None`?

Reply:
I think think this a remnant of me trying a different approach, thanks! That is a good catch.